database/xc7z/settings.sh

```diff
@@ -0,0 +1,29 @@
+# Unsetting all previous XRAY env variables
```

Comment:
Other files are named such that this should be called database/zynq7/settings.sh